docs/extensions/storage/users.md

```diff
@@ -1,12 +1,24 @@
 ---
 title: "Users"
 date: 2020-01-16T00:00:00+00:00
-weight: 35
+weight: 17
 geekdocRepo: https://github.com/owncloud/ocis
 geekdocEditPath: edit/master/docs/extensions/storage
 geekdocFilePath: users.md
 ---
 
+TODO add this to the storage overview? or is this a different part? That should be started as a separate service ? And documented elsewhere, eg. in the accounts?
+
+### User and Group provisioning
+
+In oc10 users are identified by a username, which cannot change, because it is used as a foreign key in several tables. For oCIS we are internally identifying users by a UUID, while using the username in the WebDAV and OCS APIs for backwards compatability. To distinguish this in the URLs we are using `<username>` instead of `<userid>`. You may have encountered `<userlayout>`, which refers to a template that can be configured to build several path segments by filling in user properties, e.g. the first  character of the username (`{{substr 0 1 .Username}}/{{.Username}}`), the identity provider (`{{.Id.Idp}}/{{.Username}}`) or the email (`{{.Mail}}`)
+
+{{< hint warning >}}
+Make no mistake, the [OCS Provisioning API](https://doc.owncloud.com/server/developer_manual/core/apis/provisioning-api.html) uses `userid` while it actually is the username, because it is what you use to login. 
+{{< /hint >}}
+
+We are currently working on adding [user management through the CS3 API](https://github.com/owncloud/ocis/pull/1930) to handle user and group provisioning (and deprovisioning).
```

Comment:
Shorter: "PR [user management through the CS3 API](https://github.com/owncloud/ocis/pull/1930) handles user and group provisioning and deprovisioning."